packages/aws-cdk-lib/rosetta/aws_cloudwatch/default.ts-fixture

```diff
@@ -4,6 +4,7 @@ import { Stack, Duration } from 'aws-cdk-lib';
 import * as cloudwatch from 'aws-cdk-lib/aws-cloudwatch';
 import * as route53 from 'aws-cdk-lib/aws-route53';
 import * as sns from 'aws-cdk-lib/aws-sns';
+import * as sqs from 'aws-cdk-lib/aws-sqs';
```

Comment:
I don't know about `rosetta`. What does this change do?

Reply:
It is used to translate the example in the markdown files from TypeScript to Python, Java, ... 

See: https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#rosetta